public/mobile-app/src/app.html

```diff
@@ -3,7 +3,9 @@
 	<head>
 		<title>Mobile app</title>
 		<meta charset="utf-8" />
-		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
+        <link rel="icon" href="%sveltekit.assets%/favicon.ico" sizes="32x32">
+        <link rel="icon" href="%sveltekit.assets%/icon.svg" type="image/svg+xml">
+        <link rel="apple-touch-icon" href="%sveltekit.assets%/apple-touch-icon.png">
```

Comment:
I'm not sure, but it seems like there's a whitespace issue here? Is it because of some automatic linter?